storage/range.go

```diff
@@ -155,6 +155,7 @@ type RangeManager interface {
 	Allocator() *allocator
 	Gossip() *gossip.Gossip
 	SplitQueue() *splitQueue
+	MultiRaft() *multiraft.MultiRaft
```

Comment:
Feels a bit overkill to expose `MultiRaft` for the sole purpose of determining if a store has been started. I'd prefer to remove this and do something more targeted such as adding a `Started` bool to `storage.Store`.